kbase-extension/static/kbase/js/widgets/appWidgets2/input/taxonomyRefInput.js

```diff
@@ -262,9 +243,6 @@ define([
                                 callback(taxon);
                             });
                         },
-                        formatMoreResults: function () {
```

Comment:
not sure what this was supposed to be, but I couldn't find `formatMoreResults` in the select2 docs

Reply:
Might be from an older version?